package.json

```diff
@@ -58,6 +59,10 @@
     "next-mdx-remote": "^4.4.1",
     "next-themes": "^0.2.1",
     "postcss": "~8.4.30",
+    "postcss-calc": "~9.0.1",
+    "postcss-import": "~15.1.0",
+    "postcss-mixins": "~9.0.4",
+    "postcss-simple-vars": "~7.0.1",
```

Comment:
My only thought here would be if this is needed, or if native CSS vars could be used?

Reply:
CSS variables work differently from SCSS variables and are limited/constrained. This allows us to have a Sassa-like functionality.

Reply:
Ah, okay 👍